tests/core/test_views_errors.py

```diff
@@ -40,19 +40,16 @@ def test_backoff_header_is_present_on_error_responses(self):
             self.assertEquals(response.headers['Backoff'], '10')
 
     def test_404_is_valid_formatted_error(self):
-        response = self.app.get('/unknown', status=404)
-        self.assertFormattedError(response, 404, ERRORS.MISSING_RESOURCE, "Not Found",
-                                  "The resource you are looking for could not be found.")
```

Comment:
Shouldn't the valid format of a 404 error contain the details attribute as well, now? If yes, then  I think this test should be modified to ensure that the error contains details. What do you think?